src/styles/_catalog.scss

```
@@ -1,22 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-height);
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @each $media, $cols in $media-queries {
+    @media (min-width: #{$media}) {
+      grid-template-columns: repeat($cols, 200px);
+    }
+  }
```

No need to specify (min-width: 1px) in $media-queries, you've already created grid with 1 column and in media query you should change amount of columns. Also you should use here $card-width variable
```suggestion
  @each $media, $cols in $media-queries {
    @media (min-width: #{$media}) {
      grid-template-columns: repeat($cols, $card-width);
    }
  }
```

---

**FIxed**